compiler/test/formatter_inputs/patterns.gr

```diff
@@ -26,3 +26,11 @@ match (list) {
   [_, alongidentalongident123456, _] | [_, _, alongidentalongident123456, _, _] => Some(alongidentalongident123456),
   _ => None,
 }
+
+enum Rec { Rec { x: Number, y: Number }, Tup(Number, Number) }
```

Comment:
It appears they are handled consistently to how other data type definitions are handled, moving all of the inline comments to the beginning line (#1578 fixed this behavior for expressions but not data definitions it appears). Should an issue be created to change this?

Reply:
We need an issue to fix it, but you also need to add tests to show how it'll format (even if wrong) so that we can have a diff of the correct formatting once it has been fixed.